popout.js

```diff
@@ -232,7 +232,7 @@ class PopoutModule {
       );
       /* eslint-enable no-undef */
 
-      link.on("click", () => this.onPopoutClicked(app));
+      link.on("pointerdown", () => this.onPopoutClicked(app));
```

Comment:
Hi @Posnet. So the changes in general are for the Svelte UI framework / "TRL" I have put together that is being increasingly used by 3rd party developers for package releases. So the changes discussed are to provide compatibility for all packages that use it.  This framework provides a custom Application shell using Svelte though the HTML for the app window is the same layout in regard to header buttons and such hence why there is no problem in inserting the pop out button. However, the draggable functionality of the TRL / Svelte based app header bar uses pointer events and most other input is pointer event driven. There are a couple of options depending on what you think is best to solve the issue. 

A single line solution that only has left click activating the button: 
`link.on("pointerdown", (event) => { event.stopPropagation(); if (event.button === 0) { this.onPopoutClicked(app); } });`

...or an alternate solution is to add an additional listener to prevent the pointerdown event from bubbling right after the `click` event registration:
`link.on("pointerdown", (event) => { event.preventDefault(); event.stopPropagation(); });`

Reply:
Another solution is to not use JQuery to adhoc insert the header button and instead use the `getApplicationHeaderButtons` hook to insert the button through the hook that is fired for collecting the application header buttons.

Reply:
I have resolved this part of the problem in TRL, by making the window / app drag handler in TRL ignore event targets with `header-button` class. I have added `header-button` which is the designated class added for Foundry core header buttons.